src/DatePicker/Styles.elm

```diff
@@ -1,70 +0,0 @@
-module DatePicker.Styles exposing (durationDayClasses, durationStartOrEndClasses, singleDayClasses)
-
-
-durationDayClasses : String -> Bool -> Bool -> Bool -> Bool -> Bool -> String
-durationDayClasses classPrefix isHidden isDisabled isPicked isToday isBetween =
-    let
-        hiddenClass =
-            calenderDayClass classPrefix "hidden" isHidden
```

Comment:
We should keep all the CSS classes that we used before. So that users can still use these classes to apply their custom styles if the want to. And those also help for testing purposes

Reply:
Alright, I brought back the classes – they're not _exactly_ the same since the markup changed a little, too. But it should allow the user to apply custom styles everywhere. Also I made the className prefix configurable (we need to rename it in Mercury so the global styles from older versions won't interfere here).

Reply:
Okay. But we don't need the added className prefix for that. We can wrap the existing CSS stylings with something like `:not(.new-datetime-picker)` and use that class for the DateTimePickers that we update to the new elm-css stylings. Then the CSS stylings don't apply.